lib/datadog/tracing/contrib/redis/trace_middleware.rb

```diff
@@ -0,0 +1,72 @@
+# typed: false
+
+require_relative '../patcher'
+require_relative 'ext'
+require_relative 'quantize'
+require_relative 'tags'
+
+module Datadog
+  module Tracing
+    module Contrib
+      module Redis
+        # Instrumentation for Redis
+        module TraceMiddleware
```

Comment:
What' the difference between this file and `instrumentation.rb`? They look very similar.
If they are for different versions of Redis, I recommend documenting on the module (`TraceMiddleware` here and `Instrumentation` as well).